tests/foreman/api/test_repository_set.py

```diff
@@ -23,11 +23,18 @@
 from robottelo import manifests
 from robottelo.api.utils import upload_manifest
 from robottelo.constants import PRDS, REPOSET
-from robottelo.decorators import run_in_one_thread, run_only_on, tier1, upgrade
+from robottelo.decorators import (
+    run_in_one_thread,
+    run_only_on,
+    skip_if_bug_open,
+    tier1,
+    upgrade,
+)
 from robottelo.test import APITestCase
 
 
 @run_in_one_thread
+@skip_if_bug_open('bugzilla', 1655239)
```

Comment:
By the way bug is raised for hammer CLI and I hope that is also reproducible from API. As this TC is API test.

Reply:
yes ,This is reproducible from API too.